src/components/sections/AcaPresentationSlides.tsx

```
@@ -0,0 +1,626 @@
+        </DialogContent>
+      </Dialog>
+
+      {/* Objection Modal */}
+      <Dialog open={objectionModalOpen} onOpenChange={setObjectionModalOpen}>
+        <DialogContent className="max-w-2xl">
+          <DialogHeader>
+            <DialogTitle>AI-Generated Objection Responses</DialogTitle>
+          </DialogHeader>
+          <div className="max-h-96 overflow-y-auto">
+            <div className="whitespace-pre-wrap text-sm" dangerouslySetInnerHTML={{
+              __html: objectionResponse.replace(/\*\*(.*?)\*\*/g, '<strong>$1</strong>')
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)